pkg/volume/util/recyclerclient/recycler_client.go

```diff
@@ -29,6 +29,7 @@ import (
 	"k8s.io/klog"
 )
 
+// RecycleEventRecorder is a func that defines how to record RecycleEvent.
```

Comment:
I suspect this comment could be improved but i'm not familiar enough with this code to produce something more accurate

Reply:
Thanks @BenTheElder 
Can you have a look at this PR? @saad-ali @rootfs @jingxu97 @screeley44
I'm looking forward to your suggestions!